### DISCUSSION TITLE
Load depth map into 3D point cloud, edit point cloud, export point cloud to depth map

### DISCUSSION THREAD

**Discussion by gituser123456789000** [nagadomi/nunif#273] [Ideas]
I just had this idea while looking at MoGe 3D point clouds... and I was trying to find a website I used before that allowed you to upload pictures and depth maps and see the point cloud. I haven't been able to find the site in a while, but I'll search a bit more for it.

But I had the idea of editing a point cloud to fix a depth map in 3D.. then output the improved depth map.

Is editing a point cloud possible? I don't even know...

Is outputting a point cloud to depth map possible? I don't even know...

But this would seem to be the real way to improve a depth map.. to be able to edit it via 3D point cloud. I'm sure it would be a good amount of time and work, but for single images it could be worth the effort.

**Comment by nagadomi** [comment]
I think that will end up editing the mesh.
You can edit it in Blender.

1. Import RGB image with `Images as Planes`
2. Subdivide and apply Depthmap with Displace modifier
3. Edit mesh
4. Render Z pass(Depth) with Compositor or direct render L/R image (SBS) with Stereo 3D camera

Also, `.ply` file output by MoGe can be imported in Blender.
It is a mesh data with vertex color set.

1. `File > Import > Stanford PLY`
2. `Viewport Shading > Color = Attribute`

![dog_ply](https://github.com/user-attachments/assets/0eda301d-361a-491b-a3f3-666ae98abc1a)
